src/draw.js

```diff
@@ -455,7 +455,7 @@ const Draw = Object.seal({
 	
 	message_controls: function(cb) {
 		let e = this()
-		e.firstChild.onclick = e=>cb(e, 'info')
+		e.firstChild.onclick = e=>cb(e, e.shiftKey ? 'speak' : 'info')
```

Comment:
hmm. something about this doesn't feel clean

Reply:
i think shift to speak should be moved to the mesage contents maybe, the info button is very difficult to get to and it isn't obvious that shift clicking the info button would do TTS things. it feels like a leftover hack from testing things

Reply:
> i think shift to speak should be moved to the message contents maybe,

as in shift-clicking the message text itself to speak it?

Reply:
> as in shift-clicking the message text itself to speak it?
yeah, but I thinki want to have other people's opinions on that first